website/docusaurus.config.js

```diff
@@ -4,7 +4,7 @@ const { externalLinkProcessor } = require('./tools/utils/externalLink');
 const absoluteUrl = 'http://docs-v2.apify.com';
 /** @type {Partial<import('@docusaurus/types').DocusaurusConfig>} */
 module.exports = {
-    title: 'Apify Docs v2',
+    title: 'Apify JS Client',
```

Comment:
iirc this should be the same for all of the docusaurus projects, not project specific, at least thats how i originally planned it - do you use the value in the subnav or whats the reason to change it?

Reply:
I'm currently reusing the title in the subnav as shown in Figma, but you're right - let's keep it the same for all the projects and add another `name` field in the theme options maybe?

![obrazek](https://user-images.githubusercontent.com/61918049/210061171-9580ff1f-afa5-4198-b9f6-f9dfbb860f14.png)

Reply:
Yeah that sounds better to me.